common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1348,6 +1353,11 @@
          */
         public static final String BROKER_API_REMOVE_BROKER_ACCOUNT_PATH = "/brokerApi/removeBrokerAccount";
 
+        /**
+         * URI Path constant for BrokerApi-to-Broker updateBrt request using ContentProvider.
```

Comment:
```suggestion
         * URI Path constant for Broker-to-Broker API updateBrt request using ContentProvider.
```

Reply:
This is intentional. It's the Broker (host) API to the (active) broker (process)